README.md

```diff
@@ -4,6 +4,13 @@ RTNS (RTrade Name Service) is a stand-alone IPNS record management service, desi
 
 It is essentially a modified and condensed version of [go-ipfs/namesys](https://github.com/ipfs/go-ipfs/tree/master/namesys) with minor optimizations
 
+# Setup
```

Comment:
This should be renamed to `Development`.

Other than looks good and is a much needed addition to the readme! :muscle: :rocket:

Reply:
Done.